include/boost/beast/core/string.hpp

```diff
@@ -44,6 +44,11 @@ namespace beast {
 
 namespace detail {
 
+inline string_view operator ""_sv(char const* p, std::size_t n)
```

Comment:
I like this

Reply:
This definition needs to go in **<boost/beast/core/detail/string.hpp>** (create if it doesn't exist). I use the physical organization of header files to help me know where something is, and also as a clue that something exists. I know that it is just one tiny symbol but the practice of putting an implementation detail in a public header, no matter how small, is not sustainable. Over time, there would be an accumulation of small symbols which are not indexed or locatable by a predictable physical orgnanization, leading to reduced maintainability.

Reply:
Should I move the other things in `detail` in this file as well?

Reply:
Yes, and for exactly the reason I described. I forgot that `detail::iequals` was even in the file. Looking it over, those other routines need a tidying. They use the outdated "gratuituous template" (`<class=void>`) when they should just use `BOOST_BEAST_DECL` / inline like everything else. `iequals` can probably go in **core/detail/string.ipp** and all of that jazz. `beast::iequals` can be a declaration instead of a one-line inline function that calls the implementation.